consensus/istanbul/core/roundstate.go

```diff
@@ -177,6 +182,37 @@ func (s *roundStateImpl) Round() *big.Int {
 	return s.round
 }
 
+func (s *roundStateImpl) changeRound(nextRound *big.Int, validatorSet istanbul.ValidatorSet) {
+	s.state = StateAcceptRequest
+	s.round = nextRound
+	s.desiredRound = nextRound
+
+	s.prepares = newMessageSet(validatorSet)
+	s.commits = newMessageSet(validatorSet)
+
+	// ??
```

Comment:
Are you sure this is correct?

Reply:
it seems strange, but it's the same behaviour we had before